### PR TITLE
Alex picsure 249

### PR DESCRIPTION
**Pull request by alexnktn**
Added URL param for the Application;
Added Applications dropdown;
Moved SuperAdmin options to dropdowns;
styling updates;

### CODE REVIEW

**Comment by namdets** [comment]
There are no tests. Per our new policy that was put into place just before you went on leave, you need to provide tests for any functionality that you add for both the backend and the frontend. Adding a setter call to an existing test without asserting anything is insufficient.

**Comment by alexnktn** [comment]
Adding tests now.

File: pic-sure-auth-ui/pom.xml [hunk 1]
```diff
@@ -122,7 +122,7 @@
 			<plugin>
 				<groupId>com.github.searls</groupId>
 				<artifactId>jasmine-maven-plugin</artifactId>
-				<version>3.0-alpha-01</version>
+				<version>2.2</version>
```
Comment (alexnktn) [inline pic-sure-auth-ui/pom.xml]: Could not run any tests until downgraded a version

File: pic-sure-auth-ui/src/test/javascript/headerSpec.js [hunk 2]
```diff
@@ -89,18 +98,47 @@ define(["header/header", "picSure/userFunctions", "jquery","underscore"],
...
+						.callFake(function(object, callback){
+							callback([{}]);
+						});
 						header.View.render();
 						expect($('a[href="/psamaui/userManagement"][style="visibility: hidden"]', header.View.$el).length).toEqual(1);
 					});
 					// Since it is possible that admin users would not have access to picsure, perhaps this is the wrong approach
 					// That is not an issue related to the current ticket and those discussions will not be happening now.
-					it("shows the PIC-SURE UI button without needing any specific roles", function(){
+					// it("shows the PIC-SURE UI button without needing any specific roles", function(){
```
Comment (alexnktn) [inline pic-sure-auth-ui/src/test/javascript/headerSpec.js]: unless PICSURE-UI comes by default everywhere, this should be removed.

Comment (namdets) [reply pic-sure-auth-ui/src/test/javascript/headerSpec.js]: The PIC-SURE-UI does come by default everywhere moving forward

**Pull request merged**